aodncore/pipeline/handlerbase.py

```diff
@@ -458,6 +473,19 @@ def file_type(self):
         """
         return self._file_type
 
+    @property
+    def include_regexes(self):
+        """Property to manage include_regexes attribute
+
+        :return:
+        :rtype: :py:class:`list`
+        """
+        return self._include_regexes
+
+    @include_regexes.setter
+    def include_regexes(self, regexes):
+        self._include_regexes = ensure_regex_list(regexes)
```

Comment:
It's tricky... that is what we would *usually* want, but empty string is a valid pattern *intended* to match everything.

Generic regex functions should probably follow regex conventions, otherwise it could get very confusing!

Reply:
I've update this PR and made the delete method do some additional validation to disallow the common "match everything" patterns by default.